_pages/home/release-notes/checkout-libraries-release-notes.md

```diff
@@ -0,0 +1,395 @@
+
+- Majority of old checkout imports (`@spartacus/checkout`) are now spread across the new entry points (`@spartacus/checkout/base`, `@spartacus/checkout/b2b`, `@spartacus/checkout/scheduled-replenishment`), and their secondary entry points (ex: `@spartacus/checkout/base/core`, `@spartacus/checkout/base/occ`, `@spartacus/checkout/base/components`).
+- Services are now using commands and queries instead of making use of ngrx for state management.
+- `normalizeHttpError()` is moved from the effects to the adapter layer.
+- Order confirmation components listed below are moved to the Order library.
+  - OrderConfirmationThankYouMessageComponent
+  - OrderConfirmationItemsComponent
+  - OrderConfirmationTotalsComponent
+  - GuestRegisterFormComponent
+- Place order and scheduling a replenishment business logic are moved to the order library.
+- Checkout flow has a better transition experience due to enhanced smoothing, less requests and less duplicated requests, and a uniform spinner for every step.
+- Most checkout components from Spartacus 4.0 have been re-named to be prefixed with Checkout. For example, CheckoutPlaceOrderComponent.
+
+## New functionality
+
+- `backOff` pattern is now added to the OCC adapters, and configured to retry on OCC-specific JALO errors.
+- Validation and error handling - the commands now perform various precondition checks. At the same time, commands will throw errors if the execution fails. Previously, effects were dispatching Fail actions, which usually were not handled.
+
+## Sample data changes
+
+- CheckoutShippingAddress `ContentPage` has been renamed to CheckoutDeliveryAddress
+  - title of the content page has also been changed from `Checkout Shipping Address` to `Checkout Delivery Address`
+  - page route has been also been changed from `/checkout/shipping-address` to `/checkout/delivery-address`
+- CheckoutShippingAddressComponent `CMSComponent` has been renamed to CheckoutDeliveryAddress
+- BodyContentSlot-checkoutShippingAddress `ContentSlot` has been renamed to BodyContentSlot-checkoutDeliveryAddress
+- SideContent-checkoutShippingAddress `ContentSlot` has been renamed to SideContent-checkoutDeliveryAddress
+
+Note: The new checkout library is not backwards compatible. This library is intended to be used with new applications that are created with other Spartacus 5.0 libraries. If you generate a new storefront app using schematics, the app will use the new checkout library by default. For more information, see [Integration Libraries and Feature Libraries]({{ site.baseurl }}/schematics/#integration-libraries-and-feature-libraries).
+
+## Events
+
+Checkout events are essential as they are used to cause additional side-effects. These events help remove the side effects done from `effects` when using ngrx. Events are caught either by event listeners or [queries to reset/reload data state](https://sap.github.io/spartacus-docs/commands-and-queries/#queries-overview),
+
+### CheckoutReloadQueryEvent
```

Comment:
I added this in line 64, but were you trying to mention to not include the event explanation? I think we should include them because the list of spartacus events only shows path to file

Reply:
Yep, we don't need to list them, as the list will be outdated at some point.
Customers are used to look at our source code anyways, and can see what each of the events are for.
Moreover, when we add / remove events to / from queries, we will document these changes in our migration doc, so the customers will be up to date

Reply:
removed list of events to not deal with evolving payload